commons/com.b2international.index/src/com/b2international/index/es/EsDocumentSearcher.java

```diff
@@ -190,7 +189,7 @@ public EsDocumentSearcher(EsIndexAdmin admin, ObjectMapper mapper) {
 			throw new IndexException("Couldn't execute query: " + e.getMessage(), null);
 		}
 		
-		final int totalHits = (int) response.getHits().getTotalHits();
+		final int totalHits = (int) response.getHits().getTotalHits().value;
```

Comment:
Accuracy of total hits counting changed with ES 7.0 ("Track total hits up to 10,000 by default"): https://www.elastic.co/guide/en/elasticsearch/reference/current/release-notes-7.0.0.html

Reply:
Thanks! Fixed in commit 562a6e6.